### ISSUE TITLE
 Wargus test on windows 10 2004

### ISSUE THREAD

**Issue by Epsilon93** [Wargus/wargus#332]
Hello all =)

I did several Wargus tests with the boxed battle net edition (mac / pc - not gog edition) version and a DOS version.

The data extraction of the battle net version works with wargus 2.4.1 but then there is no sound, just the music.

In addition, 2.4.1 does not propose any modification of the shaders.

So I tried with version 2.4.2 and 2.4.3 and strangely, version 2.4.2 works while version 2.4.3 does not (error message when launching wargus).

I specify that my battle net edition is in French.

Problem is the same for DOS version.

For version 2.4.2 it would be almost perfect but I have other problems which appear, the sound and the musics are ok but when I switch to opengl rendering, the games then refuse to launch.

It is therefore impossible to apply the shaders.

If I use the DOS version, the problem is the same.

During the extraction, i have some red ligne with the DOS version.

The extraction of the boxed battle net version goes without red ligne and therefore seems more reliable in terms of data.

Is there something I can do to solve my problems with version 2.4.2 and the battle net version? (config file, support for open gl rendering) I don't know anything about programming so I'm just starting out sorry!

It's very strange because on a previous test I had successfully run open gl and shaders with the DOS version. This implies that the battle net version should also work because I now have the problem with both versions (BNE and DOS).

The error message below is the one corresponding to version 2.4.2 with the BNE Edition and DOS version when I try in open gl.

The data that is extracted by the process is saved in "C:\Users\Xavier\AppData\Roaming\Stratagus" (folder "data.Wargus" - "wc2" - files "stderr" "stdout"

I hope that this feedback will help improve future versions to fully support battle net versions.

Thanks for your help!

![Message d'erreur lancement wargus 2-4-1 BNE edition open gl](https://user-images.githubusercontent.com/70840534/92323318-0cb14380-f038-11ea-8088-a79ab577bc2f.jpg)

**Comment by timfel** [comment]
Hi, and thanks for the report. Unfortunately due to life events there is currently no time for me or anyone else to do work on this, at least until the end of the year. I am sure we will get to it at some point, but it is unlikely in the short term. The new unreleased 3.0 version has mic better support for modern graphics cards and shades, so that should be fixed already in a nightly build. if you want, you can try the latest master-build (it has the version number 3.0.0, but may still have other bugs so it is unreleased).

**Comment by Epsilon93** [comment]
Hello !

Thank you very much for your reply ! I have found an alternative, I am using reshade which works great. I get a very nice result by taking the time to fine-tune the settings so I couldn't hope for better. The xbrz shader is finally forgotten ... =) Thanks again for your work, everything works perfectly so with the battle net edition mac / pc in box and wargus / stratagus 2.4.2. The games are completely revived and the main strength of the engine is of course the support for the highest resolutions (which for an rts makes the gameplay much more enjoyable) and especially the fluidity allowed by the engine when scrolling the map with the mouse! This avoids big headaches and finally allows to redo the game in good conditions. Thank you again for the work done, it is in my opinion the best version of warcraft 2 once we have managed to overcome the few installation / compatibility issues between the different versions. Good luck and continuation for the rest!
